src/Database/Migrations/2016_02_09_132439_create_permission_user_table.php

```diff
@@ -17,14 +17,15 @@ public function up()
         $table = config('roles.permissionsUserTable');
         $permissionsTable = config('roles.permissionsTable');
         $tableCheck = Schema::connection($connection)->hasTable($table);
+        $userTable = app(config('auth.providers.users.model'))->getTable();
 
         if (!$tableCheck) {
-            Schema::connection($connection)->create($table, function (Blueprint $table) use ($permissionsTable) {
+            Schema::connection($connection)->create($table, function (Blueprint $table) use ($permissionsTable, $userTable) {
                 $table->increments('id')->unsigned();
-                $table->integer('permission_id')->unsigned()->index();
-                $table->foreign('permission_id')->references('id')->on($permissionsTable)->onDelete('cascade');
+                $table->integer('role_id')->unsigned()->index();
```

Comment:
I don't get why this was changed, can you explain?

Reply:
This is indeed incorrect, it should be `permission_id` not `role_id`.
I checked my history, a mistake I made that was undone has been merged in the wrong order.

Reply:
how was this merged without proper reviewing 🤦🏽, a package for roles and permissions should insure stability for production usage.

Reply:
fixed.